lib/situation_room_web/router.ex

```diff
@@ -17,9 +17,19 @@ defmodule SituationRoomWeb.Router do
   scope "/", SituationRoomWeb do
     pipe_through(:browser)
 
-    resources("/sites", SiteController)
 
     live("/", SiteLive)
+    # sites 
+    # sites - live index
+    live("/sites", Live.Sites.Index, :index)
+    
+    # sites - crud
+    get("/sites/new", SiteController, :new)
+    post("/sites/new", SiteController, :create)
+    get("/sites/:id", SiteController, :show) #todo - make this a live view as well
+    get("/sites/:id/edit", SiteController, :edit)
+    put("/sites/:id", SiteController, :update)
+    delete("/sites/:id", SiteController, :delete)
```

Comment:
Any reason you are manually enumerating all these RESTful routes instead of just using [the `resources/4` macro](https://hexdocs.pm/phoenix/Phoenix.Router.html#resources/4)?